source/transfer_list.rst

```diff
@@ -357,8 +371,12 @@ Inputs:
 
 #. Set `tl.max_size` (`tl_base_addr + 0xc`) to `available_size`.
 
-#. Calculate the checksum as the sum of all bytes from `tl_base_addr` to
-   `tl_base_addr + tl.hdr_size`, and write the result to `tl.checksum`.
+#. Set `tl.flags` (`tl_base_addr + 0x10`) to `1` if checksums are to be used,
```

Comment:
minor: Putting the "If" clause first is more natural for engineers!

Reply:
done